src/website/templates/index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>

```

Comment:
Hi @OchiengPaul442 , thanks for this. I am just wondering whether a backend implementation should be having HTML code. Is HTML meant for the frontend repo?